keyboards/minidox/keymaps/dustypomerleau/keymap.c

```diff
@@ -312,7 +303,7 @@ void altop_finished (qk_tap_dance_state_t *state, void *user_data) {
 }
 
 void altop_reset (qk_tap_dance_state_t *state, void *user_data) {
-  switch (altop_tap_state.state) {
+  switch (td_state) {
     case SINGLE_TAP:
       unregister_code(KC_9);
```

Comment:
I think you can actually use `register_code16(S(KC_9))` and the `unregister_code16` function for these,

Reply:
Oh wow thanks - will test it that way. I wasn't even aware that `register_code16()` existed!